distrobox-create

```diff
@@ -166,9 +166,20 @@ generate_command() {
 
 # First, check if the image exists in the host
 if ! podman image exists "${container_image}"; then
-	echo "Image not found, run this command first:"
-	echo "	podman pull ${container_image}"
-	exit 1
+    echo >&2 "Image not found."
+    echo >&2 "Do you want to pull the image now?[y/n]"
+    read -r yn
+    case $yn in
+        [Yy]* )
+            podman pull ${container_image}
+            break
+            ;;
+        [Nn]* )
```

Comment:
please use just

n | N  | no ) 

to be more precise with the input (like replying nYes will count as no)